pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -353,3 +467,59 @@ func (c *K8sOrchestrator) IsFSSEnabled(ctx context.Context, featureName string)
 	log.Debugf("cluster flavor %q not recognised. Defaulting to false", c.clusterFlavor)
 	return false
 }
+
+// GetVolumeAnnotations fetches annotations from PVC bound to passed volumeID and returns
+// annotation key-value pairs as a map
+func (c *K8sOrchestrator) GetVolumeAnnotations(ctx context.Context, volumeID string) (map[string]string, error) {
```

Comment:
GetVolumeAnnotations is misleading==> Rename to GetPVCAnnotations ?

Reply:
Since we're adding this methods to CO agnostic interface, and PVC is purely a kubernetes concept, will it be ok to use PVC in our naming?
These methods are being declared in interface and invoked from controller.

Reply:
Isn't term "annotation" also k8s specific? If so, maybe we can have methods that returns exactly what the callers need, for ex: `IsFakeAttachAllowed`

Reply:
Changed the interface methods to `IsFakeAttachAllowed` & `ClearFakeAttached`.